src/trace_processor/importers/proto/proto_trace_tokenizer.h

```diff
@@ -164,8 +164,9 @@ class ProtoTraceTokenizer {
       }
 
       // If there's not enough bytes in the reader, then we cannot do anymore.
-      size_t size_incl_header = hdr_size + field_size;
-      if (size_incl_header > avail) {
+      uint64_t size_incl_header = hdr_size + field_size;
```

Comment:
Instead of this, I suggest writing the code as follows to avoid the overflow entriely:
```
      // If there's not enough bytes in the reader, then we cannot do anymore.
      if (field_size > avail - hdr_size) {
        return base::OkStatus();
      }
      size_t size_incl_header = hdr_size + field_size;
```

Reply:
makes sense, thanks!